src/main/scala/org/deepdive/inference/datastore/SQLInferenceDataStore.scala

```diff
@@ -839,67 +660,53 @@ trait SQLInferenceDataStore extends InferenceDataStore with Logging {
         if (isFixed || weightlist == ""){
           execute(s"""DROP TABLE IF EXISTS ${weighttableForThisFactor} CASCADE;
```

Comment:
DROP TABLE is such a toxic operation that we should consider making it a method that one has to call in order to generate the SQL.  Inside that method, we would have operations that ensure we are only dropping tables inside the DeepDive namespace.  I am worried that a typo in a tablename somewhere yields a name clash with user tables, and then catastrophe